lua/autorun/server/sv_cfc_loadout_init.lua

```diff
@@ -4,5 +4,6 @@ util.AddNetworkString( "CFC_Loadout_InitialSpawn" )
 util.AddNetworkString( "CFC_Loadout_SendRestrictions" )
 include( "cfc_loadouts/server/sv_cfc_loadout.lua" )
 AddCSLuaFile( "cfc_loadouts/client/cl_cfc_functions.lua" )
-AddCSLuaFile( "cfc_loadouts/client/cl_cfc_interface.lua" )
-AddCSLuaFile( "cfc_loadouts/client/cl_cfc_loadout.lua" )
\ No newline at end of file
+-- AddCSLuaFile( "cfc_loadouts/client/cl_cfc_interface.lua" )
```

Comment:
Could just remove this if the file doesn't exist anymore

Reply:
Sure thing, although I will say that It never existed...